cmd/conformance-tests/run.sh

```diff
@@ -18,19 +18,19 @@ set -xeuo pipefail
 
 cd $(dirname $0)
 
-cd $(go env GOPATH)/src/github.com/kubermatic/kubermatic
+cd $(go env GOPATH)/src/k8c.io/kubermatic
```

Comment:
let's use relative path here like in other shell scripts

Reply:
Fixed.